oasis/cloud-migration-tool.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: default
+description: >-
+  Migrating data from bare metal servers to the cloud with minimal downtime
+---
+# Cloud Migration Tool
+
+{{ page.description }}
+{:class="lead"}
+
+The `arangosync-migration` tool allows you to easily move from on-premises to 
+the cloud while ensuring a smooth transition with minimal downtime.
+Start the cloud migration, let the tool do the job and, at the same time,
+keep your local cluster up and running. 
+
+Some of the key benefits of the cloud migration tool include:
+- Safety comes first - pre-checks and potential failures are carefully handled.
+- Your data is secure and fully encrypted.
+- Ease-of-use with a live migration while your local cluster is still in use.
+- Get access to what a cloud-based fully managed service has to offer: 
+high availability and reliability, elastic scalability, and much more.
+
+## Prerequisites 
+
+Before getting started, make sure the following prerequisites are in place:
+
+- Go to [Oasis Cloud](https://cloud.arangodb.com/home){:target="_blank"}
+and sign in. If you don’t have an account yet, sign-up to create one.
+
+- Generate an Oasis API key and API secret. See a detailed guide on 
+[how to create an API key](api-getting-started.html#creating-an-api-key).
+
+### Setting up the target deployment in Oasis
+
+Continue by [creating a new Oasis deployment](deployments.html#how-to-create-a-new-deployment)
+or choose an existing one.
+
+The target deployment in Oasis requires specific configuration rules to be
+set up before the migration can start:
+
+- **Configuration settings**: The target deployment must be compatible with the
+source data cluster. This includes the ArangoDB version that is being used,
+the DB-Servers count, and disk space.
+- **Deployment region and cloud provider**: Choose the closest region to your
+data cluster. This factor can speed up your migration to the cloud.
+
+After setting up your Oasis deployment, wait for a few minutes to become 
+fully operational.
+
```

Comment:
### Limitations
Currently only clusters are supported (question to arangosync team, for bothe source and target, or target only?) This means that moving data towards an Oasis developer mode deployments isn't supported.

Reply:
Are there any restrictions on platform where to run the tool (Linux, Windows, ARM?)?

Reply:
### external access
Your on-prem deployment need to be accessible form the outside world... remark about firewall on port 8629 would be nice imo

Reply:
@Robert-Stam  @nerpaula 
1) on limitations - the tool already checks for Developer mode indeed. Maybe it's worth adding to docs too.
2) No platform restrictions for now, should run anywhere. But we haven't tested it on Windows yet - arangosync itself wasn't tested on Windows for a long time too. It is on the list. The binary will have '0.1.0-preview-1' version for now.  Maybe we should let the customers know that it is still a beta / release candidate, not the stable version.
3) external access - there is already remark in documentation: `Make sure the host and port are available for the Oasis platform`. But maybe it is worth mentioning firewall explicitly, too.

Reply:
Added notes about clusters-only, developer mode not supported, and firewall on port 8629. Also added a note to inform users that the documentation describes the preview version and is subject to change.